undertow/src/main/java/org/wildfly/extension/undertow/HttpListenerService.java

```diff
@@ -57,13 +60,16 @@
 
     private final ChannelUpgradeHandler httpUpgradeHandler = new ChannelUpgradeHandler();
     protected final InjectedValue<ListenerRegistry> httpListenerRegistry = new InjectedValue<>();
+    // remove this once other subsystems convert to capabilities
```

Comment:
Perhaps the field should be @Deprecated in that case to let other subsystems know.

Reply:
Thanks. Done.